laos_gggi/combine_data.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import sys
+
+sys.path.append("..")
+from emdat_processing import process_emdat
+from world_bank_data_loader import download_wb_data
+from GPCC_data_loader import download_gpcc_data
+from co2_processing import process_co2
+from ocean_heat_processing import load_ocean_heat
+
+
+def final_data():
+    # Create the dictionary that contains the combined files
+    merged_dict = {}
+
+    # 1. EM-DAT data representing number of events per year (index: Year, ISO3)
+    emdat = process_emdat()
+    merged_dict["emdat_events"] = emdat["df_prob_filtered_adjusted"].drop(
+        columns=["Region", "Subregion"]
+    )
+
+    # 2. EM-DAT data representing the event damages (index: Year, ISO3)
+    merged_dict["emdat_damage"] = emdat["df_inten_filtered_adjusted"].drop(
+        columns=["Country", "Region"]
+    )
+
+    # 3. The WB data, index (Year, ISO3)
+    merged_dict["wb_data"] = download_wb_data()
```

Comment:
Need to check that the ISO3 codes between EMDAT and WB agree. I had to do some alignment between the shapefiles and the EMDAT data, so I suspect it will be true for the WB/EMDAT data as well.

Reply:
Yes, you are right.

I used the code you used previously. I adjusted and I checked the results. It worked well